sync/clients/__init__.py

```diff
@@ -59,8 +65,8 @@ def _send_request(self, request: httpx.Request) -> httpx.Response:
 
     async def _send_request_async(self, request: httpx.Request) -> httpx.Response:
         try:
-            for attempt in Retrying(
-                stop=stop_after_attempt(3),
+            async for attempt in AsyncRetrying(
```

Comment:
Yes. You need to use asyncretrying as the retrying class uses time.sleep instead of asyncio.sleep. Time.sleep will block the async event loop and prevent all other async stuff from happening, which is bad.